COLLABORATOR_GUIDE.md

```diff
@@ -75,47 +77,86 @@ Collaborators or additional evidence that the issue has relevance, the
 full responsibility for the change, defer to another Collaborator.
 
+If you are the first collaborator to approve a pull request that has no CI yet,
+please start one (see [testing and CI](#testing-and-ci) for further information
+on how to do that) and post the link to the CI in the PR. Please also start a
+new CI in case a code change was done due to a addressed review comment or a
+rebase.
+
+In case there are already enough approvals (`LGTM`), a CI run and the PR is open
+longer than the minimum waiting time without any open comments, please do not
+just add another approval. Instead go ahead and land the PR after checking the
+CI outcome.
+
+### Consensus Seeking
+
+If there is no disagreement amongst Collaborators, a pull request should be
+landed given appropriate review, a green CI and the minimum
+[waiting time](#waiting-for-approvals) for a PR. If it is still awaiting the
+[minimum time to land](#waiting-for-approvals), please add the `ready` label to
+it so it is obvious that the PR can land as soon as the time ends.
+
+Where there is discussion amongst Collaborators, consensus should be sought if
+possible. The lack of consensus may indicate the need to elevate discussion to
+the TSC for resolution.
+
 If any Collaborator objects to a change *without giving any additional
 explanation or context*, and the objecting Collaborator fails to respond to
 explicit requests for explanation or context within a reasonable period of
 time, the objection may be dismissed. Note that this does not apply to
 objections that are explained.
 
-For non-breaking changes, if there is no disagreement amongst
-Collaborators, a pull request may be landed given appropriate review.
-Where there is discussion amongst Collaborators, consensus should be
-sought if possible. The lack of consensus may indicate the need to
-elevate discussion to the TSC for resolution (see below).
-
-Breaking changes (that is, pull requests that require an increase in
-the major version number, known as `semver-major` changes) must be
-[elevated for review by the TSC](#involving-the-tsc).
-This does not necessarily mean that the PR must be put onto the TSC meeting
-agenda. If multiple TSC members approve (`LGTM`) the PR and no Collaborators
-oppose the PR, it can be landed. Where there is disagreement among TSC members
-or objections from one or more Collaborators, `semver-major` pull requests
-should be put on the TSC meeting agenda.
+Note that breaking changes (that is, pull requests that require an increase in
+the major version number, known as `semver-major` changes) must be [elevated for
+review by the TSC](#involving-the-tsc). This does not necessarily mean that the
+PR must be put onto the TSC meeting agenda. If multiple TSC members approve
+(`LGTM`) the PR and no Collaborators oppose the PR, it should be landed. Where
+there is disagreement among TSC members or objections from one or more
+Collaborators, `semver-major` pull requests should be put on the TSC meeting
```

Comment:
Suggestion: `should be` -> `may be`. Doing nothing and deferring to the status quo (closing the PR) is certainly a thing that happens.